.github/workflows/nv-h100.yml

```diff
@@ -52,8 +52,8 @@ jobs:
         run: |
           unset TORCH_CUDA_ARCH_LIST # only jit compile for current arch
           cd tests
-          python -m pytest $PYTEST_OPTS -n 4  unit/ --torch_ver="2.0" --cuda_ver="12"
-          python -m pytest $PYTEST_OPTS -m 'sequential' unit/ --torch_ver="2.0" --cuda_ver="12"
+          python -m pytest $PYTEST_OPTS -n 4  unit/ --torch_ver="2.0" --device_ver="12"
```

Comment:
This probably isn't the right wording we want for this - @mrwyattii for thoughts on how to better handle this, since this test is clearly only running on cuda so a cuda version makes sense here.